src/app/cube/new-home/learning-object-info/learning-objects/learning-objects.component.ts

```diff
@@ -1,15 +1,96 @@
 import { Component, OnInit } from '@angular/core';
+import { LearningObject, User } from '@entity';
+import { FeaturedObjectsService } from 'app/core/featuredObjects.service';
+import { LearningObjectService } from 'app/core/learning-object.service';
+import { UserService } from 'app/core/user.service';
+import { UsageStatsService } from 'app/cube/core/usage-stats/usage-stats.service';
+
 
 @Component({
   selector: 'clark-learning-objects',
   templateUrl: './learning-objects.component.html',
   styleUrls: ['./learning-objects.component.scss']
 })
 export class LearningObjectsComponent implements OnInit {
+  featuredObject: LearningObject; // the learning object to display
+  numReleasedObjects = 0; // default number of released objects before the service provides a new number
+
+  constructor(private featureService: FeaturedObjectsService,
+              private learningObjectService: LearningObjectService,
+              private userService: UserService,
+              private usageStatsService: UsageStatsService) { }
 
-  constructor() { }
+  async ngOnInit(): Promise<void> {
+    await this.usageStatsService.getLearningObjectStats().then(stats => {
+      this.numReleasedObjects = Math.floor(stats.released / 10) * 10;
+    });
+    await this.featureService.getFeaturedObjects();
+    await this.featureService.featuredObjects.subscribe(objects => {
+      this.featuredObject = objects[1];
+    });
+    await this.learningObjectService.fetchLearningObjectWithResources({
+      author: this.featuredObject.author._name,
+      cuidInfo: {
+        cuid: this.featuredObject.cuid
+      },
+      id: this.featuredObject.id
+    }, ['outcomes']).subscribe((object: LearningObject) => {
+      this.featuredObject = object;
+    });
+  }
+
+  /**
+   * Displays the featured object's levels, formatted
+   *
+   * @returns A comma separated string of learning object levels
+   */
+  displayFeaturedObjectLevels() {
+    if(this.featuredObject?.levels.length === 1) {
+      return this.featuredObject.levels[0];
+    } else {
+      let levels = '';
+      this.featuredObject.levels.forEach((level, index, array) => {
+        if(index === array.length - 1) {
+          levels += ' and ' + level;
+        } else {
+          levels += level + ', ';
+        }
+      });
+      return levels;
+    }
+  }
+
+  /**
+   * Parses the featured object's description from HTML to text
+   *
+   * @returns The description string
+   */
+  displayDescription() {
```

Comment:
We can also stick this as innerHTML so the tags can take affect for bold and italics .